### ISSUE TITLE
Improve suggestions

### ISSUE THREAD

**Issue by GambaJo** [dslul/openboard#43]
I use the german dictionary. I type the word "Ich" (which means "I", a very common word, that I used a lot) and this are the suggestions I got:
- Och
- Sich
- Mich

I can bring many other examples:
Starts to type the word "Test" --> "Tes":
- Tee
- Ted
- Tea

In other keyboard apps I often only to start the first 2 or 3 chars, and can build whole sentences without typing any car key. Only by selecting one suggestion after another. This is very fast.

- Auto-correction off
- Suggestions on
- Personalised Suggestions on

**Comment by jwsp1** [comment]
Agree, it should rather suggest word completion instead of looking for typos and replacing already written characters.
